### PR TITLE
Initial version of tbb env in build

### PR DESCRIPTION
**Pull request by napetrov**
# Description
In this change we enable initial version of windows ci

Changes proposed in this pull request:
- Fix download scripts on windows
- Enable NuGet package download for tbb
- Cleanup makes from 32 bit targets
- Update install instructions

### CODE REVIEW

File: .ci/env/tbb.bat [hunk 1]
```diff
@@ -0,0 +1,67 @@
+@echo off
+rem ============================================================================
+rem Copyright 2018-2020 Intel Corporation
```
Comment (averbukh) [inline .ci/env/tbb.bat:3]: 2020

File: INSTALL.md [hunk 2]
```diff
@@ -100,21 +101,21 @@ For details, see [System Requirements for oneDAL](https://software.intel.com/con
 
 8. Build oneDAL via command-line interface. Choose the appropriate commands based on the platform and the compiler you use:
```
Comment (outoftardis) [inline INSTALL.md:102]: I'd add here that not only the platform and the compiler matter but also the interfaces (DAAL vs. oneAPI)

File: INSTALL.md [hunk 3]
```diff
@@ -19,6 +19,7 @@
 
 Required Software:
 * C/C++ Compiler
+* DPC++ Compiler
```
[the remark this answers is not in the document]
Comment (napetrov) [reply INSTALL.md:22]: Any. Although there is only intel implementation only at the moment. But assuming there will be another one implemented - this should work

[the remark this answers is not in the document]
Comment (outoftardis) [reply INSTALL.md:22]: sure

File: INSTALL.md (hunk 2 above)
Comment (napetrov) [reply INSTALL.md]: fixed

**Review by outoftardis — APPROVED** [review]

**Review by homksei — APPROVED** [review]

**Pull request merged**